Formula/fsql.rb

```diff
@@ -0,0 +1,20 @@
+class Fsql < Formula
+  desc "Search through your filesystem with SQL-esque queries."
+  homepage "https://github.com/kshvmdn/fsql"
+  url "https://github.com/kshvmdn/fsql/archive/v0.1.1.tar.gz"
+  sha256 "d3ff000b8374d2eb9a44aa20220c758a82a24c82ee8a369fae146e84fc422f2c"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/kshvmdn").mkpath
+    ln_s buildpath, buildpath/"src/github.com/kshvmdn/fsql"
+    system "go", "build", "-o", bin/"fsql"
+  end
+
+  test do
+    output = shell_output(bin/"fsql -version")
```

Comment:
Sorry, I think I'm misunderstanding; the tests seem to be passing now? Do you mean that they should be testing functionality?